src/memberdef.cpp

```diff
@@ -5523,24 +5536,56 @@ void MemberDefImpl::setAnonymousUsed() const
 
 bool MemberDefImpl::hasCallGraph() const
 {
-  return m_impl->hasCallGraph;
+  if (m_impl->hasCallGraph & Entry::CALL_SHOW) return true;
```

Comment:
I would think these these 4 lines can be done in one line like so
```c
return m_impl->hasCallGraph & (Entry::CALL_SHOW|Entry::CALL_CONFIG)
```

Reply:
No I don't think so as in case:
- CALL_SHOW = false
- CALL_HIDE = true  so explicit hide setting
- CALL_CONFIG = true so default is show
in the 4 line case the CALL_HIDE is honored, but in the reduced 1 line form it isn't .